src/components/ui/AntDesign.jsx

```diff
@@ -1,6 +1,6 @@
 import React, {useState} from 'react';
 import { Avatar, Button, Cascader, Col, DatePicker, Input, Layout, Link, List, Row, Space, Tag, TextArea, TimePicker, Typography } from 'antd';
-import 'antd/dist/antd.css';
+/* import 'antd/dist/antd.css'; */
```

Comment:
Why did you comment this out? Is there some kind of incompatibility between the two different frameworks across components?

Reply:
Yes, it seems to apply on all other libraries if not commented out.

Reply:
Ah well that is problematic. It didn't seem to cause problems when merging grommet though.

Reply:
The changes are minor, mainly margins on text, so if you want I can "uncomment" it out.